packages/metro-file-map/src/lib/dependencyExtractor.js

```diff
@@ -14,26 +14,23 @@
 'use strict';
 
 const NOT_A_DOT = '(?<!\\.\\s*)';
-const CAPTURE_STRING_LITERAL = (pos /*: number */) =>
+const CAPTURE_STRING_LITERAL = (pos: number) =>
```

Comment:
I believe this is a file that *needs* to be CommonJS, though interestingly making this change did not cause test failures beyond the shape of the exports changing from using `export default` instead of `module.exports`